holoviews/selection.py

```diff
@@ -170,8 +220,26 @@ def _expr_stream_updated(self, hvobj, selection_expr, bbox):
 
 class link_selections(_base_link_selections):
     selection_expr = param.Parameter(default=None)
-    unselected_color = param.Color(default="#99a6b2")  # LightSlateGray - 65%
-    selected_color = param.Color(default="#DC143C")  # Crimson
+    unselected_color = param.Color(default="#e6e9ec")
+    selected_color = param.Color(default=None, allow_None=True)
```

Comment:
I think these parameters need docstrings. E.g. how is None interpreted for selected_color?

Reply:
Done